doc/src/build/fullnode.md

```diff
@@ -288,3 +288,119 @@ If you followed the instructions for [Building from Source](#building-from-sourc
     ```
 
 Your Full node starts on: `http://127.0.0.1:9000`.
+
+# Pruning 
+
+## Object Pruning
+
+Object pruning involves the removal of older object versions from the fullnode db. 
+New object versions are added to the db during transaction execution (making older versions ready for garbage collection) and can become a 
+problem if not pruned, as it can slow down the performance of the db and take up too much disk space. 
+Object pruning works in the background by tailing checkpoint stream, identifying each
+checkpoint which contains object versions which are now eligible for pruning.
+
+A sui node can enable object pruning by adding this `authority-store-pruning-config` config:
```

Comment:
```suggestion
You can enable pruning for a Sui node by adding the `authority-store-pruning-config` config:
```
Please specify the name and location of the file to add it to

Reply:
added `fullnode.yaml`